recipes/r-jobqueue/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = '1.3.1' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-jobqueue
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/jobqueue_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/jobqueue/jobqueue_{{ version }}.tar.gz
+  sha256: b26c9577f86edfb688f44cc645f1fa18621aa553897f3a4563abcfd86b7568ce
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-r6
+    - r-cli
+    - r-later
+    - r-magrittr
+    - r-parallelly
+    - r-promises
+    - r-ps
+    - r-rlang
+    - r-semaphore
+  run:
+    - r-base
+    - r-r6
+    - r-cli
+    - r-later
+    - r-magrittr
+    - r-parallelly
+    - r-promises
+    - r-ps
+    - r-rlang
+    - r-semaphore
+
+test:
+  commands:
+    - $R -e "library('jobqueue')"           # [not win]
+    - "\"%R%\" -e \"library('jobqueue')\""  # [win]
+
+about:
+  home: https://cmmr.github.io/jobqueue/index.html
+  dev_url: https://github.com/cmmr/jobqueue
+  license: MIT
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
```

Comment:
For clarity, this should stay here unchanged. It will be sourced from the CRAN tarball.